services/121-service/src/payments/fsp-integration/intersolve-voucher/services/intersolve-voucher-cron.service.ts

```diff
@@ -67,7 +67,7 @@ export class IntersolveVoucherCronService {
     }
 
     // Get the first intersolve programFinancialServiceProviderConfigurationId that has intersolveVoucherWhatsapp as FSP
-    // ##TODO: store the programFspConfigurationId or the usename and password in the intersolveRequest table so we know which credentials to use for the cancelation
```

Comment:
This never worked, an we also never get complaints about this, so leaving this for an IntersolveVoucher refactor...